api/controllers/OpenEAppController.js

```diff
@@ -92,7 +97,7 @@ const OpenEAppController = {
             ),
             documents: casebookResponse.documents,
             originalCost: HelperService.formatToUKCurrency(
-                casebookResponse.payment.netAmount
+                casebookResponse.payment.transactions[0].amount
```

Comment:
could casebookResponse.payment.transactions be unset here?  Would recommend a guard condition needed here with a fallback value - 0?  Also formatToUKCurrency probably covers it but worth checking if it always include two decimal places.

Reply:
Yep for sure, luckily `formatToUKCurrency` uses the [internatianlization api](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) to format numbers so that should take care of the two decimal places 🤞